language/nb.json

```diff
@@ -0,0 +1,301 @@
+{
+  "semantics": [
+    {
+      "label": "Oppgavebeskrivelse",
+      "description": "Beskriv oppgava di her."
+    },
+    {
+      "label": "ord",
+      "widgets": [
+        {
+          "label": "Standard"
+        }
+      ],
+      "entity": "Ord",
+      "field": {
+        "label": "Ord",
+        "fields": [
+          {
+            "label": "Hint",
+            "description": "Hint til svaret."
+          },
+          {
+            "label": "Svar",
+            "description": "Svar til hintet."
+          },
+          {
+            "label": "Ektra hint"
```

Comment:
Mener du "Ekstra"?

Reply:
Ja, beklager. Det er en skrivefeil. Du har rett, det skal være "Ekstra".
Yes sorry. It is a typo. You are correct, it should be "Ekstra".

I can close this PR and submit a new one if you like?